ider/src/ider.component.spec.ts

```diff
@@ -40,9 +40,9 @@ describe('IderComponent', () => {
     expect(component.ider).toBeInstanceOf(AMTIDER)
     expect(component.redirector).toBeInstanceOf(AMTRedirector)
 
-    expect(component.redirector?.server).toEqual('testServer')
-    expect(component.redirector?.authToken).toEqual('testToken')
-    expect(component.redirector?.host).toEqual('testDeviceId')
+    expect(component.redirector?.server?.toString()).toEqual('testServer')
```

Comment:
we shouldn't have to toString if we define them as strings (even thought they are inputs)